.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: node_js
-sudo: false
+services:
+- docker
+sudo: required
 node_js:
 - '4.2'
 script:
 - npm test
+- make docker-build
+deploy:
+  provider: script
+  script: scripts/deploy.sh
```

Comment:
@krancour would it make sense to just let DockerHub build the image on a post-commit hook?